sdks/python/apache_beam/internal/gcp/auth.py

```diff
@@ -119,7 +122,7 @@ class _Credentials(object):
 
   @classmethod
   def get_service_credentials(cls, pipeline_options):
-    # type: (PipelineOptions) -> Optional[google.auth.credentials.Credentials]
+    # type: (PipelineOptions) -> Optional[_ApitoolsCredentialsAdapter]
```

Comment:
What caused the return type change?

Reply:
The return type didn't change, I just updated the type hint to be accurate.